### PR TITLE
Remove webpush and vapid

### PR DESCRIPTION
**Pull request by otterbotter**
This PR removes VAPID and Webpush code from the project.

When this PR is merged, the following should be done:
- [ ] Remove VAPID environment variables from backend.yml in ansible-config
- [ ] Remove VAPID secrets from password store for each environment

Questions:
- [x] Should the `notiifcations` app be removed as a part of this PR? Is it related to the webpush stuff?

### CODE REVIEW

File: muni_portal/core/admin.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 from django.contrib import admin
 from muni_portal.core.models import Webhook, WebPushSubscription, WebPushNotification, WebPushNotificationResult
```
Comment (jbothma) [inline muni_portal/core/admin.py:2]:
```suggestion
from muni_portal.core.models import Webhook
```

Comment (jbothma) [reply muni_portal/core/admin.py:2]: could you remove these models too?

Comment (otterbotter) [reply muni_portal/core/admin.py:2]: oh thanks! i'll do that and include migrations for them

**Review by jbothma — APPROVED** [review]

**Comment by otterbotter** [comment]
I missed the `py-webpush` package - I assume that also needs to be removed.

I'll open a new PR that removes it.

**Pull request merged**